pyproject.toml

```diff
@@ -118,12 +118,14 @@ test = [
     ## herdingspikes
     "herdingspikes<=0.3.99",
 
-
     ## sliding_nn
     "pymde",
     "torch",
     "pynndescent",
-    
+
+    ## IBL
+    "ibl-neuropixel",
```

Comment:
Why do we need this ?

Reply:
For testing against the IBL implementation. Check the tests